### DISCUSSION TITLE
How to display totals per group on a Grouped Stacked Bar Chart

### DISCUSSION THREAD

**Discussion by 22367rh** [apexcharts/apexcharts.js#4021] [Q&A]
I was using the chart style from https://apexcharts.com/javascript-chart-demos/bar-charts/grouped-stacked-bars/ as I noticed it fit my needs perfectly except I cannot get it to show the total per group per category. This is the modification to the default example but that ends up as shown in the screenshot attached. 

Anyone know a way to achieve what I need or should I raise a feature request?

`plotOptions: {
          bar: {
            horizontal: true,
			dataLabels: {
			  total: { 
				enabled: true, 
				offsetX: 10,
				style: { fontSize: '13px', fontWeight: 900 } 
			  }
			}
          }
        }`

![image](https://github.com/apexcharts/apexcharts.js/assets/360784/b2c0e02b-3c24-4e44-b1e9-74d9ff08b298)

**Comment by 23tux** [comment]
@22367rh Did you ever find a solution to this? I'm facing the exact same problem

**Comment by shivaliSingh13** [comment]
@22367rh Have you resolved this? I am facing the same issue.

**Comment by sulminnim** [comment]
The issue of not displaying Group total in apexchart has been raised as an issue
https://github.com/apexcharts/apexcharts.js/issues/4173

It may be solved one day, but I couldn't wait, so I solved it as follows.
https://codepen.io/sulmin/pen/gOyBxzB

**Comment by batyr993** [comment]
Anyone found a solution for this in 2025?

**Comment by vingerha** [comment]
I am not sure if this works for all series but there is already a w.globals.stackedSeriesTotals. 
Noting that I am not an avid codepen user, example: https://codepen.io/vingerha/pen/Byoroyb
